keyboards/takashicompany/minidivide/info.json

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
Please run `qmk format-json` on this file.

Reply:
Thanks.

I fixed. [d18fce1](https://github.com/qmk/qmk_firmware/pull/19870/commits/d18fce114e53102ae547e64ed9c6a3f98951c8a5)